zebra-scan/src/config.rs

```diff
@@ -41,6 +55,9 @@ impl Default for Config {
     fn default() -> Self {
         Self {
             sapling_keys_to_scan: IndexMap::new(),
+            listen_addr: None,
+
+            // TODO: Add a const generic for specifying the default cache_dir path, like 'zebra' or 'zebra-scan'?
```

Comment:
@oxarbitrage I think this TODO would solve the panic. I'd prefer to do this in a follow-up PR if that's okay, I don't think the bug was introduced in this PR.

Reply:
Ok, fair enough. Thanks.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/8241/commits/3ca3af0ec5e491e9ef4f52e52b3d821b711faf1c.